docs/docs/how_to.md

```diff
@@ -67,6 +67,7 @@ class MyRobot:
             "x": unpacked[3 * N_ROBOTS],
             "y": unpacked[3 * N_ROBOTS + 1]
         }
+        data["is_goal"] = unpacked[3 * N_ROBOTS + 2]
```

Comment:
@Adman is there a serious reason why leaving `3 * N_ROBOTS` would be better than abstracting it out in here? Asking mostly out of curiosity :)

Thanks!

Reply:
Yeah, I'll change the docs too make it more clear.

Reply:
Done.